carbon-design-system/carbon-addons-iot-react
# 2278 table column title is clipped

Closes #2278

**Summary**
Small change but plenty of testing. Increased column header label line hight to prevent text from being cut of

**Change List (commits, features, bugs, etc)**
- Increased label line hight to prevent text from being cut of
- Updated visual snap since the text was repositioned slighty

**Acceptance Test (how to verify the PR)**
1. Go to https://deploy-preview-2981--carbon-addons-iot-react.netlify.app?path=/story/1-watson-iot-table-statefultable--simple-stateful-example
2. Zoom out to 90%
3. Verify that Column header is not vertically clipped
4. include different table sizes and knob configurations and make sure the text is properly aligned and not clipped

**Regression Test (how to make sure this PR doesn't break old functionality)**
- All table stories looks good




**Things to look for during review**
